src/cmd-buildextend-legacy-oscontainer

```diff
@@ -20,6 +20,19 @@ sys.path.insert(0, cosa_dir)
 from cosalib import cmdlib
 
 
+parser = argparse.ArgumentParser()
+parser.add_argument("--arch-tag", help="append arch name to push tag",
+                    action='store_true')
+parser.add_argument("--name", help="oscontainer name",
+                    action='store', default=f'{name}')
```

Comment:
`default=f'{name}'` doesn't make sense here any longer because `name` is defined later. Would it make sense to move the commit that deletes `--name` before this commit?